tests/run_tests.sh

```diff
@@ -20,28 +20,19 @@ STATUS=0
 
 export HDF5_DISABLE_VERSION_CHECK=1
 
-# Run the standard set of unit tests, producing JUnit compatible output
-coverage run -m pytest tests --junitxml=tests/reports/test-results-"$(date +'%Y%m%d%H%M%S')".xml
-RESULT=$?
 
-if [ "$RESULT" -ne "0" ]; then
-    STATUS=1
-    echo "ERROR: unittest generated errors"
-fi
+# Exit status used to report back to caller
+STATUS=0
 
-echo "\n"
-echo "Test Coverage Estimates"
-coverage report --omit="tests/*"
-coverage html --omit="tests/*" -d tests/coverage
+# Run the standard set of unit tests, producing JUnit compatible output
+pytest --cov=hybig --cov=harmony_service \
+       --cov-report=html:reports/coverage \
+       --cov-report term \
+       --junitxml=reports/test-reports/test-results-"$(date +'%Y%m%d%H%M%S')".xml || STATUS=1
```

Comment:
Not sure if STATUS=1 is set from code above.

Reply:
This is where we set it when the test fails so that the exit code is non-zero on failures.